nlpsandboxclient/evaluation.py

```diff
@@ -0,0 +1,232 @@
+#!/usr/bin/env python3
+# Author: Yao Yan
+# Description: This script is used to evaluate the annotation by participants with
+# gold standards, we will conduct the annotation based on subcategory: date, person name,
+# physical address
+from abc import ABCMeta
+import json
+import re
+
+
+# take as input the location of
+class Evaluation(metaclass=ABCMeta):
+    """Evaluate the different types"""
+    gs_dict_seq = dict()
```

Comment:
General comments about variable names:
- do not use abbreviations whenever possible to make the code more readable. Here an external developer would be confused with name like "gs" and "sys" at the time of the first reading.
- do not include the type of the variable in the variable name (shorten the variable name)